### PR TITLE
story (SC-774): RMDC datalogger signals

### PR DESCRIPTION
**Pull request by gordon-epc**
## Jira Story

[SC-774](https://epcpower.atlassian.net/browse/SC-774)

## About

Provide way to alias datalogger signals. Store the mapping in `*.epc` file.

## Associated Pull Requests

*   [ ] [grid-tied](https://github.com/epcpower/grid-tied/pull/314)
*   [ ] [st](https://github.com/epcpower/st/pull/38)

## Update Changelog

*   [x] Changelog updated

## Validation

See ST PR.

### CODE REVIEW

**Review by epcAnker — APPROVED** [review]

File: epyqlib/device.py [hunk 1]
```diff
@@ -361,6 +363,7 @@ def _load_config(
             "access_password_path",
             "ParameterQuery;FactoryAccess;FactoryAccess",
         )
+        d.setdefault("software_hash_path", None)
```
Comment (bholewin-epc) [inline epyqlib/device.py:366]: Not a fan of a variable named `d`, I understand it predates this ticket.

**Review by bholewin-epc — APPROVED** [review]

File: epyqlib/device.py (hunk 1 above)
Comment (gordon-epc) [reply epyqlib/device.py]: Renamed in b988885fb3c9d284410ca9c1f83a0a902bb2aaeb

**Review by davidpmichaud — APPROVED** [review]
This is a good improvement 👍. Can we take a further action to migrate the other "magic" symbols (read derogatorily) in EPyQ to also use this model? For example, "StatusBits" is a special frame name, as is "SaveToEE_command" and "SaveToEE_status". There may be others. @gordon-epc

**Comment by gordon-epc** [comment]
> This is a good improvement 👍. Can we take a further action to migrate the other "magic" symbols (read derogatorily) in EPyQ to also use this model? For example, "StatusBits" is a special frame name, as is "SaveToEE_command" and "SaveToEE_status". There may be others. @gordon-epc

We can, but it's going to be a different PR.

**Pull request merged**